app/input_listener.py

```diff
@@ -71,7 +71,7 @@ def __init__(self):
         process.start()
 
     def start_listening(self, queue):
-        def on_movement(self):
```

Comment:
This doesn't seem necessary, did you mean to commit this?

Reply:
yes this a correction from the documentation